### PR TITLE
Jae/collegetown ui

### PR DESCRIPTION
**Pull request by connorreinhold**
Implemented UI for campus eateries, design of the pill, and the framework for the menu popup that will appear upon tapping on a collegetown eatery.

This is CollegeTownMenuActivity - currently completely copied from CampusMenuActivity (just renamed) - this is the big thing left to get done
Screen Shot 2019-04-06 at 11 02 44 PM

This is still the MainListFragment just with additional code for CollegeTownEateries

Screen Shot 2019-04-12 at 12 18 29 PM

This PR addresses #100

### CODE REVIEW

**Comment by ningning621** [comment]
Just a reminder to make issues for `CampusMenuActivity` and the pill when you merge this PR!

**Comment by jyc979** [comment]
From what I discussed with Conner, we're not planning to add Menu for CT eateris any time soon so he wanted me to disable the menu expansion upon clicking ct eateries. Let's not have the CtMenuActivity just yet we can put it back when the backend work is done and can create a separate issue for it.

**Comment by connorreinhold** [comment]
^^Updated these! Fixed pill designs too

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
             android:name="com.google.android.geo.API_KEY"
             android:value="@string/google_maps_key"/>
         <activity
```
Comment (ningning621) [inline app/src/main/AndroidManifest.xml]: Let's remove this for now?

File: app/src/main/java/com/cornellappdev/android/eatery/LoginFragment.java [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import android.support.v4.app.Fragment;
 import android.support.v4.app.FragmentTransaction;
 import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/LoginFragment.java]: remove

File: app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java [hunk 3]
```diff
@@ -9,7 +9,10 @@
 import android.support.design.widget.BottomNavigationView;
 import android.support.v4.app.FragmentTransaction;
 import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java]: remove

File: app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java [hunk 4]
```diff
@@ -80,6 +89,7 @@ public void successLogin(BrbInfoQuery.AccountInfo accountInfo) {
                 @Override
                 public void onPageFinished(WebView view, String url) {
                     // Gets executed on every page redirect. Only want to evaluate JS once
+                    Log.i("Loading page",url);
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java]: remove

File: app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java [hunk 5]
```diff
@@ -98,30 +108,30 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                                 getSupportFragmentManager().beginTransaction();
                         switch (item.getItemId()) {
                             case R.id.action_home:
-                                transaction.replace(R.id.frame_fragment_holder,
-                                        new MainListFragment())
+                                transaction
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java]: nice!

File: app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java [hunk 6]
```diff
@@ -86,12 +113,46 @@ public void initializeButtons(View rootView) {
...
-            mButtons.put(id, tempButton);
+            mCampusButtons.put(id, tempButton);
             tempButton.setOnClickListener(this);
             changeButtonColor(R.color.blue, R.color.wash, tempButton);
         }
     }
 
+    public void initializeCollegeTownEateryButtons(View rootView) {
+        int[] viewIds = {
+                R.id.american,
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java]: are these temporary? or predetermined by backend?

File: app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java [hunk 7]
```diff
@@ -86,12 +113,46 @@ public void initializeButtons(View rootView) {
...
+                R.id.korean,
+                R.id.chinese,
+                R.id.japanese,
+                R.id.indian};
+        for (int id : viewIds) {
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java]: add newline above

**Comment by ningning621** [comment]
> ^^Updated these! Fixed pill designs too

@connorreinhold  can you update the description with screenshots of the new pill designs?

File: app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java (hunk 6 above)
Comment (connorreinhold) [reply app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java]: These were just fillers I added the rest of them and just committed

**Pull request merged**